wikiclue/src/routes/change-username/+page.svelte

```diff
@@ -3,9 +3,29 @@
   import Overlay from "../../components/overlay.svelte";
   import { writable } from "svelte/store";
   import { goto } from '$app/navigation';
+  import { authHandlers, authStore } from "../../store/store";
+  import { onMount } from "svelte";
+  import { auth } from "../../firebase/firebase";
+
 
   // When we have the database set up with users, this would need to occur onload
-  const currentUsername = writable("Mr Kenny");
+  let currentUsername = writable(null);
+    onMount (async () => {
+    	authStore.subscribe(async ({ user }) => {
+			  if (!user) {
+				  console.error('Unable to grab user id');
+				  return;
+			  }
+
+			  const uid = user.uid;
+        if(user){
+          const username = await authHandlers.getUsername(uid);
+          currentUsername.set(username);
+          }
+            // Would need to use this functionality to pre load levels data with words from database
+            // depending on if the user selects easy medium or hard
+      });
+    });
 
   const changeWasClicked = writable(false);
   let newUsername = "";
```

Comment:
I know this isn't the changes in your PR but could we have all global constants for the file at the top of the page just under the imports?